packages/core/core-flows/src/cart/workflows/update-line-item-in-cart.ts

```diff
@@ -73,6 +78,11 @@ export const updateLineItemInCartWorkflow = createWorkflow(
       return [item.variant_id].filter(Boolean)
     })
 
+    const setPricingContext = createHook("setPricingContext", {
+      cart,
```

Comment:
question: Should we also hard-code the currency code and region ID here?

Reply:
Done